lib/i18n/index.ts

```diff
@@ -0,0 +1,41 @@
+import { po } from 'gettext-parser';
+import Gettext from 'node-gettext';
+import { logger } from '../logger';
+import { readSystemFile } from '../util/fs';
+
+export const gt = new Gettext();
```

Comment:
better export a wrapper function for `gt.gettext` instead of export the whole instance.

name could be `gt` to be short, as @rarkins suggested